src/abhijitMJun20/Indexofuse.java

```diff
@@ -0,0 +1,35 @@
+package abhijitMJun20;
+
+public class Indexofuse {
+	void indexuseof(String name){
+		for(int index=0;index<name.length();index++){
+			char ch= name.charAt(index);
+			if(index==name.indexOf(ch)) {
```

Comment:
Here you should add one more if a condition that is (name.indexOf(ch) != name.lastIndexOf(ch))